### DISCUSSION TITLE
Help me to understand about Loco deployment process

### DISCUSSION THREAD

**Discussion by sidoqi** [loco-rs/loco#268] [Q&A]
Hi guys,

I want to discuss about following statement from [Deployment step](https://loco.rs/docs/getting-started/deployment/).

>And copy your binary along with your config/ folder to the server. You can then run myapp start on your server.

Why we still need to copy the `config` folder? Can we just use the binary? My concern is that the config file can prone to error or get manipulated that might affect the deployment. Can we just serialize it as one binary with the app? This will really help to minimize the unintended issues. Thanks!

**Comment by kaplanelad** [comment]
Hey @jurukode 
This is a good point. Currently, the configuration not included as part of the binary.  and the proccess serch the file in config file in order the environment that the proccess get. 

You can open an issue for that and this is something that we can to implement.

**Reply by sidoqi** [reply]
Added here: https://github.com/loco-rs/loco/issues/270

**Comment by kaplanelad** [comment]
great

**Comment by sidoqi** [comment]
It seems the reason for the current arrangement is because of this

> Change is easy - often you want to try different values under different loads in production, with Axum only, you have to recompile, redeploy. With Loco you can set a config and restart the process.

source: https://loco.rs/docs/getting-started/axum-users/
